src/pkg/selectors/scopes_test.go

```diff
@@ -513,3 +513,20 @@ func (suite *SelectorScopesSuite) TestScopeConfig() {
 		})
 	}
 }
+
+func (suite *SelectorScopesSuite) TestDiscreteCopy() {
+	var (
+		t     = suite.T()
+		orig  = stubScope(AnyTgt)
+		clone = discreteCopy(orig, "fnords")
+	)
+
+	for k, v := range orig {
+		if k != rootCatStub.String() {
+			assert.Equal(t, v.Target, clone[k].Target)
+		} else {
+			assert.Equal(t, AnyTgt, v.Target)
+			assert.Equal(t, "fnords", clone[k].Target)
+		}
+	}
```

Comment:
could you please also add an assertion to make sure the original has not changed as well. I know it seems obvious, but sometimes assumptions like these are what come back to get you in the end

Reply:
The assertion on 528 is precisely that.

Reply:
🤦‍♀️